packages/transactions/src/coders/transaction.test.ts

```diff
@@ -38,7 +40,78 @@ describe('TransactionCoder', () => {
     expect(JSON.stringify(decoded)).toEqual(JSON.stringify(transaction));
   });
 
-  it('Can encode TransactionCreate', () => {
+  it('Can encode TransactionScript with inputs, outputs and witnesses', () => {
```

Comment:
I think we can add some data on all these 0 values as this will not validate but only parse is good for testing.

gasPrice, gas limit, maturity, lengths etc...

Reply:
Sounds good, I'll update the transactions.  What about witnesses, input, and outputs?  I think it's a bit overkill for these because they have separate coders, but I can update these here or as a follow-up issue.

Reply:
Maybe random values would be better than = values for each field. But it already looks great.

If we could also generate some random witnesses data would be nice also.